### ISSUE TITLE
Transfer to yourself

### ISSUE THREAD

**Issue by krlosMata** [hermeznetwork/wallet-ui#404]
- When a user performs a transfer to its ethereum address, a `send` transaction appears in the list of transactions but there is no `received` transactions. This situation leads that balance does not match with transactions performed.

- I think this is an edge case but still wallet should handle this user flow

**Comment by elias-garcia** [comment]
Right! I've already commented this in the past, I think that a user shouldn't be able to send funds to himself. I don't think that any user wants to send funds to himself paying a fee. The solution for me would be to disable the button in the form to prevent this behavior. What do you think? Am I missing some use case where a user of the wallet wants to do this? @krlosMata @AlbertoElias

**Comment by AlbertoElias** [comment]
@jbaylina wants us to support this for when people use this for testing purposes

**Comment by mmilenkovic** [comment]
@AlbertoElias  Can we mark this one as `Won't fix` or we keep it open?

**Comment by AlbertoElias** [comment]
@mmilenkovic We have to leave it open
